ingest.yaml

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
I understand this file is a sample anyway, but just to make sure I understand the purpose: this filed should better be put into the `services/` directory right?